guides/common/modules/proc_adding-errata-to-a-host.adoc

```diff
@@ -0,0 +1,13 @@
+[id="Adding_errata_to_a_single_host_{context}"]
+= Adding errata to a single host
+
+Use the following procedure to add errata to a host.
+
+.Procedure
+. In the Satellite web UI, navigate to *Hosts > All Hosts* and select the host you want to add Errata to.
+. On the new Host page, select the *Content* tab.
+. On the *Content* page select the *Errata* tab.
+. Select the errata you want to add to the host or select the checkbox at the top of the list to add all installable errata.
+Click the check box next to any errata you wish to remove from a full list.
+. Using the three dots icon of the errata you want to add to the host, select *Apply via Katello agent* if UNKNOWN, *Apply via Remote Execution* if UNKNOWN, or *Apply via customized remote execution* if UNKNOWN.
```

Comment:
katello agent and remote execution are two different technologies that can be used to achieve the task. REX is preferred, but katello-agent is the only option for users, who don't have connectivity from Capsule to the target host via SSH. The Customized REX mean, it will use REX but will allow user to customize the execution, e.g. set a specific time when it should be applied.